Update 20-pids.md (#159)
NFDI4Microbiota/nfdi4microbiota-knowledge-base: docs/_Research-Data-Management/20-pids.md

For example, Mathias Mueller is a fairly common name in Germany. However, with a PID we can identify the correct researcher and link them with all of their work. Just like a unique finger print a PID distinguishes you from other researchers and allows you to connect your ID with your professional information including affiliations, publications, grants, and peer reviews. {% cite gonzalez_rdm %}

## DOI
Digital Object Identifier or DOI is another form of persistent identifier. Papers, articles, and published datasets may have a DOI which links these items. This makes it easier to find these digital objects online. Even if the publisher changes the DOI of the published article, dataset, etc. will not change.

# Benefits of assigning PIDs
+ Increase visibility
+ Connects the author with the work
+ You can use ORCID to login to other applications (ie. [Coscine](https://docs.coscine.de/en/))
+ Machine readable
+ Adheres to FAIR principle
+ Permanent link to digital object

# Use cases

> PIDs are seen as the first step towards making research FAIR through increasing the findability. A case study looks at the next step in this FAIRification process by linking PIDs with metadata. This will allow for linking digital resources that are assigned PIDs together. According to Cousijn, the PID Graph establishes connections between different entities within the research landscape, thereby enabling both researchers and institutions to access new information. [Cousijn2021](https://www.sciencedirect.com/science/article/pii/S2666389920302440?via%3Dihub)

Reviewer: Since you mentioned the name of the first authors already, the standard approach to citation is to [suppress the author name](https://github.com/inukshuk/jekyll-scholar#suppressing-author-names).

Adding an URL (that is actually not the doi here) is redundant with the bibliography that should provide this link.
```suggestion
PIDs are seen as the first step towards making research FAIR through increasing the findability. A case study looks at the next step in this FAIRification process by linking PIDs with metadata. This will allow for linking digital resources that are assigned PIDs together. According to Cousijn and collaborators, the PID Graph establishes connections between different entities within the research landscape, thereby enabling both researchers and institutions to access new information {% cite --suppress_author Cousijn2021 %}.
```

# Link to the FAIR data principles

> This [article](https://www.nature.com/articles/sdata201618) in Scientific data discusses the FAIR principles for research data management. It is the first formal publication of the FAIR Principles, and includes the rationale behind them, and some exemplar implementations in the community. { % cite wilkinson_2016 % }

Reviewer: Regarding the delivery:
- Same comment as above regarding the URL.

Regarding the content:
- I know that PIDs help pave the way towards FAIR data but it is not clear from the text the link with the PID.
- feel free to link to the FAIR page of the knowledge base ;) 
```suggestion
Wilkinson and collaborators discuss the FAIR principles for research data management in the first formal publication of the principles, they include the rationale behind them, and some exemplar implementations in the community { % cite --suppress_author wilkinson_2016 % }.
```

## DOI
Digital Object Identifier or DOI is another form of persistent identifier. Papers, articles, and published datasets may have a DOI which links these items. This makes it easier to find these digital objects online. Even if the publisher changes the DOI of the published article, dataset, etc. will not change.

# Benefits of assigning PIDs
+ Increase visibility
+ Connects the author with the work
+ You can use ORCID to login to other applications (ie. Coscine)
+ Machine readable
+ Adheres to FAIR principle
+ Permanent link to digital object

# Use cases

> [doi: 10.1016/j.patter.2020.100180](https://doi.org/10.1016/j.patter.2020.100180)

Reviewer: thanks for these edits, could you please put the reference in a context, the doi is a bit lonely here.

I would be even better to add the doi to the https://github.com/NFDI4Microbiota/nfdi4microbiota-knowledge-base/blob/main/docs/_bibliography/references.bib file.

# Link to the FAIR data principles

> [doi: 10.1038/sdata.2016.18](https://doi.org/10.1038/sdata.2016.18)

Reviewer: same comment as above
